src/app/assets/styles/_color_variables.scss

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Global colors
+$primary: #326DE6;
+$line: #AAAAAA;
+$subline: #888888;
+
+// Replica Set Details related colors
+$replicasetdetails-sidebar-bg: #FAFAFA;
```

Comment:
Move all things that are specific only to replica set details page to its css. Follow the rule: "I want to be able to delete replica set detail feature folder to delete all things related to it." 

Also, I'm not sure whether location of this file is correct and its purpose. Please add a todo and create an issue to decide where to put global style variables and how to generally do global css changes.